
00:37:49 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:39:41 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:39:41 - main - ERROR - <module>: 224 - cannot write empty image as JPEG
Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 221, in <module>
    sending()
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 190, in sending
    bot.send_photo(chat_id=channel_id, photo=page)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/__init__.py", line 1825, in send_photo
    apihelper.send_photo(
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 469, in send_photo
    files = {'photo': util.pil_image_to_file(photo)}
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG
